### PR TITLE
UIEvent types implementation

### PR DESCRIPTION
**Pull request by ddenisyuk**
UIEvent types implementation: MouseEvent, TouchEvent, FocusEvent, KeyboardEvent, WheelEvent, InputEvent

### CODE REVIEW

File: src/worker-thread/index.ts [hunk 1]
```diff
@@ -87,6 +95,12 @@ const globalScope: GlobalScope = {
   SVGElement,
   Text,
   Event: WorkerDOMEvent,
+  MouseEvent: MouseEvent,
+  TouchEvent: TouchEvent,
+  FocusEvent: FocusEvent,
+  KeyboardEvent: KeyboardEvent,
+  WheelEvent: WheelEvent,
+  InputEvent: InputEvent,
```
Comment (malec-palec) [inline src/worker-thread/index.ts]: Fix linter's errors here:
```
ERROR: 98:3    object-literal-shorthand  Expected property shorthand in object literal ('{MouseEvent}').
ERROR: 99:3    object-literal-shorthand  Expected property shorthand in object literal ('{TouchEvent}').
ERROR: 100:3   object-literal-shorthand  Expected property shorthand in object literal ('{FocusEvent}').
ERROR: 101:3   object-literal-shorthand  Expected property shorthand in object literal ('{KeyboardEvent}').
ERROR: 102:3   object-literal-shorthand  Expected property shorthand in object literal ('{WheelEvent}').
ERROR: 103:3   object-literal-shorthand  Expected property shorthand in object literal ('{InputEvent}').
```

File: src/worker-thread/index.amp.ts [hunk 2]
```diff
@@ -93,6 +101,12 @@ const globalScope: GlobalScope = {
   SVGElement,
   Text,
   Event: WorkerDOMEvent,
+  MouseEvent: MouseEvent,
+  TouchEvent: TouchEvent,
+  FocusEvent: FocusEvent,
+  KeyboardEvent: KeyboardEvent,
+  WheelEvent: WheelEvent,
+  InputEvent: InputEvent,
```
Comment (malec-palec) [inline src/worker-thread/index.amp.ts]: The same linter's errors as in `src/worker-thread/index.ts`

File: src/worker-thread/index.ts (hunk 1 above)
Comment (ddenisyuk) [reply src/worker-thread/index.ts]: done

File: src/worker-thread/index.amp.ts (hunk 2 above)
Comment (ddenisyuk) [reply src/worker-thread/index.amp.ts]: done

**Pull request merged**